airflow/config_templates/config.yml

```diff
@@ -2350,6 +2350,14 @@ triggerer:
 kerberos:
   description: ~
   options:
+    mode:
+      description: |
+        Specify the mode in which kerberos container will run, can be sidecar mode vs init container mode.
+        The sidecar mode runs indefinitely but the init container mode performs kinit once and exits.
+      version_added: 2.7.3
```

Comment:
I assume this is correct

Reply:
It should be in kuberneters configs does it not?

Reply:
It should be gone altogether.

Reply:
It's conveted to a `--mode` switch now.

Reply:
Correct, removing it